README.md

```diff
@@ -12,14 +12,19 @@ HttpKernel adapter for use of Symfony and Laravel frameworks with PHP-PM. See ht
 
           composer require php-pm/httpkernel-adapter:dev-master
 
-> **Note**: For Symfony, make sure your `AppKernel` is autoloaded in your
-> `composer.json` (shouldn't be an issue for projects created using the Standard
-> Edition after November 2015):
->
-> ```
-> {
->     "autoload": {
->         "classmap": ["app/AppKernel.php"]
->     }
-> }
-> ```
+**Symfony 4**:
+
+Make sure `httpkernel` can access the Symfony session service. Add this to `services.yml`:
+
+	session.handler:
+	    public: true
```

Comment:
Does it work? I've tried it on symfony 4 and only solution that works was - https://github.com/php-pm/php-pm-httpkernel/issues/60#issuecomment-357061010

Reply:
after adding these lines, following error appears:
```
In CheckDefinitionValidityPass.php line 60:
                                                                                                                                                                                                                    
  The definition for "session.handler" has no class. If you intend to inject this service dynamically at runtime, please mark it as synthetic=true. If this is an abstract definition solely used by child definit  
  ions, please add abstract=true, otherwise specify a class to get rid of this error.    
```

Reply:
Could you check if declaring the service as synthetic works? See https://symfony.com/doc/current/service_container/synthetic_services.html for how to.